crowbar_framework/app/views/barclamp/dns/_edit_attributes.html.haml

```diff
@@ -3,7 +3,12 @@
     = header show_raw_deployment?, true
 
   .panel-body
-    = string_field :domain
+
+    %fieldset
+      = string_field :domain, :disabled => "disabled"
```

Comment:
@cmurphy I think you are right that we never see this not being deployed, but maybe there could be ways (if not now, than in future) to deploy the initial crowbar stack differently. So I'd say @Itxaka's proposal makes sense